spec/models/facility_spec.rb

```diff
@@ -30,23 +30,6 @@
     it { should delegate_method(:follow_ups_by_period).to(:patients).with_prefix(:patient) }
   end
 
-  context ".assigned_patients" do
-    let!(:assigned_facilities) { create_list(:facility, 2) }
-    let!(:registration_facility) { create(:facility) }
-    let!(:patients) do
-      [create(:patient, assigned_facility: assigned_facilities.first, registration_facility: registration_facility),
-        create(:patient, assigned_facility: assigned_facilities.second, registration_facility: registration_facility)]
-    end
-
-    it "returns assigned patients for facilities" do
-      expect(Facility.where(id: assigned_facilities).assigned_patients).to match_array patients
-    end
-
-    it "ignores registration patients" do
-      expect(Facility.where(id: registration_facility).assigned_patients).to be_empty
-    end
-  end
```

Comment:
Should we reinstate specs for `assigned_hypertension_patients`, or remove that scope from `Facility`?

Reply:
Will reinstate the `assigned_hypertension_patients` specs with a shoulda matcher. 👍